connectors/jetty-connector/src/test/java/org/glassfish/jersey/jetty/connector/ProxyTest.java

```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
There seems to be something wrong. This file was not modified in 2020 and yet the copyright plugin requires a copyright change.

Reply:
It is because the plugin removed the section, and this is considered as a new change.
 * Contributors:
 *    Marcelo Rubim